addons/l10n_nl/data/account.account.template.csv

```diff
@@ -185,7 +187,7 @@
 "4040","Kantinekosten","4040","account.data_account_type_expenses","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_5","False"
 "4041","Bedrijfskleding","4041","account.data_account_type_expenses","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_5","False"
 "4042","Overige reiskosten","4042","account.data_account_type_expenses","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_5","False"
-"4043","""Congressen, seminars en symposia""","4043","account.data_account_type_expenses","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_5","False"
+"4043","Congressen, seminars en symposia","4043","account.data_account_type_expenses","l10n_nl.l10nnl_chart_template","l10n_nl.account_tag_5","False"
```

Comment:
useless diff

Reply:
I don't think so, it is displayed as "Congressen, seminars en symposia" (with the brackets) and it was asked by the partner who provided the feedback that spawned this task.

Reply:
Account name with quotes looks weird to me: to check

Reply:
What I meant was that this account was displayed as "Congressen, seminars en symposia" with the quotes before this PR. So this change removes the quotes. I don't really know why it had quotes in the first place - maybe a typo.